submission/Kartik-NITP-2020-09-26.ipynb

```diff
@@ -0,0 +1,977 @@
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "C3wRTspr9vh_",
+        "outputId": "cdc1048c-7cca-4bf7-bc1b-b90210e5ca0e"
+      },
+      "source": [
+        "np.zeros(10,)"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0.])"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 12
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nwEPij5F9viD"
+      },
+      "source": [
+        "#### Create a Numpy 1d array of 10 ones\n",
+        "\n",
+        "`array([ 1.,  1.,  1.,  1.,  1.,  1.,  1.,  1.,  1.,  1.])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "efl7_9aK9viE",
+        "scrolled": true,
+        "outputId": "eb6b7a42-cba5-45dd-f980-a592442693e1"
+      },
+      "source": [
+        "np.ones(10,)"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "array([1., 1., 1., 1., 1., 1., 1., 1., 1., 1.])"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 13
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qm18HslT9viJ"
+      },
+      "source": [
+        "#### Create a Numpy 1d array of 10 fives\n",
+        "\n",
+        "`array([5, 5, 5, 5, 5, 5, 5, 5, 5, 5])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "wPDfBofF9viL",
+        "outputId": "85b01977-01a7-4339-cda8-ccec9a02fc97"
+      },
+      "source": [
+        "np.ones(10,dtype=np.int)*5"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "array([5, 5, 5, 5, 5, 5, 5, 5, 5, 5])"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 16
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WPvaR37T9viO"
+      },
+      "source": [
+        "#### Create a Numpy 1d array consisting of all the integers from 10 to 50\n",
+        "\n",
+        "`array([10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26,\n",
+        "       27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43,\n",
+        "       44, 45, 46, 47, 48, 49, 50])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "6QmtwvU79viP",
+        "outputId": "5ec799f1-39a6-4b89-a64b-dca721d2d1f9"
+      },
+      "source": [
+        "np.array(range(10,50))"
```

Comment:
**Create a Numpy 1d array consisting of all the integers from 10 to 50**

`np.array(range(10,51))`